gui/src/components/Taipy/tableUtils.tsx

```diff
@@ -128,7 +128,7 @@ export const baseBoxSx = { width: "100%" };
 export const paperSx = { width: "100%", mb: 2 };
 export const tableSx = { minWidth: 250 };
 export const headBoxSx = { display: "flex", alignItems: "flex-start" };
-export const iconInRowSx = { fontSize: "body2.fontSize" };
+export const iconInRowSx = { gridColumnStart: 2, fontSize: "body2.fontSize" };
```

Comment:
Did you check the impacts of this change ?
- Add a row icon
- Aggregate icon
- Table filter icon
- Table Filter list row Validate and Delete icons
- Table value edit icons

Reply:
- Add a row icon: fixed
- Aggregate icon: fixed 
- Table filter icon: check
- Table Filter list row Validate and Delete icons: fixed
- Table value edit icons: check

I took this occasion to improve the filter popover style too.